spec/model/relation_spec.rb

```diff
@@ -176,6 +209,27 @@ class User < Foo::Model; end
       end
     end
 
+    context 'when other scopes are present' do
```

Comment:
didn't notice a difference running these specs w/ the previous code. would be fine removing this block and sticking with the coverage you added above.